### DISCUSSION TITLE
React Native Stack Navigation - slide from top

### DISCUSSION THREAD

**Discussion by wherewego-nina** [software-mansion/react-native-screens#1498] [Ideas]
Is there a reason there is no option to animate navigation transition from top to bottom ? ("slide_from_top"). Has anyone found another way to do this?

**Comment by tboba** [comment]
Hi, @wherewego-nina! I believe when we've been planning stack animations we haven't thought about the case of sliding the screen from top. There also might be some reasons why it has not been planned (like maybe weird slide animations from this direction?) But I believe it's a good field to discover. If you feel strong enough, you might try to implement it by itself (any help is really appreciated!) and submit a PR with the change!

For now, I'll move this discussion to the Ideas category.
